functions/parseCommand.js

```diff
@@ -20,11 +20,10 @@ exports.getPrefix = (client, msg) => {
   let prefix = msg.guildConf.prefix;
   const escape = client.funcs.regExpEsc;
   if (prefix instanceof Array) {
-    prefix.forEach((pref) => {
+    prefix.some((pref) => {
```

Comment:
ESLint fail 
  23:24  error    Expected to return a value in arrow function  array-callback-return